pkg/sqlite/driver_cgo.go

```diff
@@ -75,7 +78,13 @@ func (c *CustomSQLiteConn) Close() error {
 	return conn.Close()
 }
 
+var driverLogShown atomic.Bool
+
 func createDBConn(dbPath string, disableForeignKeys bool) (*sqlx.DB, error) {
+	if driverLogShown.CompareAndSwap(false, true) {
+		logger.Debug("SQLite: using the CGo driver")
```

Comment:
I think we could do with a debug message that logs the entire connection URL, and that would be a perfect place to add in whether the cgo/non-cgo driver is being used. It also would be able to run on every connection.

Reply:
I can do that